File: C/arrrev.c
#include <stdio.h>
int main(){
int s;
scanf("%d",&s);
int arr[s];
for(int i=0;i<s;i++){
    scanf("%d",&arr[i]);
}

int temp;
for(int i=0;i<s/2;i++){
    temp=arr[i];
    arr[i]=arr[s-i-1];
    arr[s-i-1]=temp;
}
for(int i=0;i<s;i++){
    printf("%d ",arr[i]);
}



return 0;
}

File: JavaPW/arrrev.java
public class arrrev {

    static void swap(int a[], int i, int j) {
        int temp = a[i];
        a[i] = a[j];
        a[j] = temp;
    }

    static void reverse(int arr[]) {
        int i = 0, j = arr.length - 1;
        while (i < j) {
            swap(arr, i, j);
            i++;
            j--;
        }
    }

    public static void main(String[] args) {
        int arr[] = {55, 88, 99, 77, 55, 60, 22, 10, 78, 65};
        reverse(arr);
        for (int i = 0; i < arr.length; i++) {
            System.out.println(arr[i]);
        }
    }
}
public class arrrev {

    static void swap(int a[], int i, int j) {
        int temp = a[i];
        a[i] = a[j];
        a[j] = temp;
    }

    static void reverse(int arr[]) {
        int i = 0, j = arr.length - 1;
        while (i < j) {
            swap(arr, i, j);
            i++;
            j--;
        }
    }

    public static void main(String[] args) {
        int arr[] = {55, 88, 99, 77, 55, 60, 22, 10, 78, 65};
        reverse(arr);
        for (int i = 0; i < arr.length; i++) {
            System.out.println(arr[i]);
        }
    }
}
